cli/cmd/install.go

```diff
@@ -56,6 +56,7 @@ type installConfig struct {
 	ProxyAutoInjectEnabled           bool
 	ProxyInjectAnnotation            string
 	ProxyInjectDisabled              string
+	ProxyLogLevel                    string
```

Comment:
since `installConfig` already contains a `proxyConfigOptions`, which itself contains a `proxyLogLevel`, is adding the field directly to `installConfig` necessary?

Reply:
Unless I'm missing something, `installConfig` doesn't contain `proxyConfigOptions`. That's `installOptions`

Reply:
Ah you're right! I misread this.